specification/network/.azure-pipelines/nrp-swagger-validation.yml

```diff
@@ -0,0 +1,32 @@
+# Starter pipeline
```

Comment:
Could we move it to another forlder, for example `.azure-pipelines/network-validation.yml` or `network-scripts/azure-pipelines.yml`. We would like to use `specification/` folder only for specs.

Reply:
@sergey-shandar done